sources/packages/web/src/components/aest/institution/modals/AssessOfferingChangeModal.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <modal-dialog-base
+    :showDialog="showDialog"
+    @dialogClosed="dialogClosed"
+    :title="title"
+    :subTitle="subTitle"
+  >
+    <template v-slot:content>
+      <div class="mt-2">
+        <v-form ref="offeringChangeApprovalForm" v-model="isFormValid">
+          <v-textarea
+            v-model="assessmentNotes"
+            variant="outlined"
+            label="Notes"
+            :rules="[(v) => !!v || 'Notes is required']"
+            required
+          ></v-textarea>
+        </v-form>
+      </div>
+    </template>
+    <template v-slot:footer>
+      <footer-buttons
+        :primaryLabel="submitLabel"
+        @primaryClick="submitForm"
+        @secondaryClick="dialogClosed"
+      />
+    </template>
+  </modal-dialog-base>
+</template>
+
+<script lang="ts">
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+import { useModalDialog } from "@/composables";
+import { computed, ref } from "vue";
+import { OfferingStatus, VForm } from "@/types";
+import { OfferingChangeAssessmentAPIInDTO } from "@/services/http/dto";
+
+export default {
+  components: { ModalDialogBase },
+  props: {
+    offeringStatus: {
```

Comment:
To avoid passing properties that would not belong to the component and are more likely to be part of the `showModal` method, I `overrode` the `showModal` in the below example.
Maybe would be worth following the same.
![image](https://user-images.githubusercontent.com/61259237/182475048-175a242e-351a-4038-99b6-bfc58da4416f.png)